examples/using-guidebot-flags.md

```diff
@@ -44,44 +44,48 @@ However, you can use the switch case with the neat little `message.flags` concep
 
 ## Using with Arguments
 
-Let's say you're trying to make a command to add or take a role to/from a user. This can be consolidated into one command using the `message.flags` method. This command is based off of the [Guidebot](https://github.com/AnIdiotsGuide/guidebot) framework, and is called role.
+Let's say you're trying to make a command to add or take a role to/from a user. This can be consolidated into one command using `message.flags`. This command is based off of the [Guidebot](https://github.com/AnIdiotsGuide/guidebot) framework, and is called role.
 
 ```js
 exports.run = (client, message, args, level) => { // eslint-disable-line no-unused-vars 
   switch (message.flags[0]) {
     // This would be 'role -add'.
-    case ('add'): {
+    case 'add': {
       // Check if the message mentions a user.
-      if (message.mentions.users.size === 0) return message.reply('Please mention a user to give the role to.');
+      if (!message.mentions.users) return message.reply('Please mention a user to give the role to.');
       const member = message.guild.member(message.mentions.users.first());
       // This is the name of the role. F`or example, if you do 'role -add @York#2400 The Idiot Himself', the name of the role would be 'The Idiot Himself'.
       const name = args.slice(1).join(' ');
       // Find the role on the guild.
       const role = message.guild.roles.find('name', name);
       // End the command if the bot cannot find the role on the server.
       if (!role) return message.reply('I can\'t seem to find that role.');
-      member.addRole(role).catch(e => {
-        return message.channel.send(`Something went wrong! **Error:**\n${e}`);
-      });
-      message.channel.send(`I've added the ${name} role to ${message.mentions.users.first().username}.`);
+      try {
+        member.addRole(role);
+        message.channel.send(`I've added the ${name} role to ${member.dsiplayName}.`)
+      } catch (e) {
+        throw e;
```

Comment:
This is purely redundant. And these errors aren't caught as you're not `await`ing these promises.